### PR TITLE
VAL 3019 queries and database layer

### PR DESCRIPTION
**Pull request by kiss90benedek**
With one database for categories

### CODE REVIEW

File: categories/static/js/create.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            .always(submitResponseHandler);
+    }
+
+    function submitSuccessHandler(response) {
+        $serverSuccess.text('Category created successfully');
+        $serverSuccess.show();
+    }
+
+    function submitErrorHandler(response) {
+        $serverFailure.text('Something went wrong. Maybe this category already exists');
```
Comment (fadani) [inline categories/static/js/create.js:42]: Then you can use Hungarian error messages as well. :)

**Comment by fadani** [comment]
👍

File: categories/static/js/create.js (hunk 1 above)
Comment (kiss90benedek) [inline categories/static/js/create.js:42]: Oh well. :D It will be hard to get used to it.

Good point, thanks!

Comment (kiss90benedek) [inline categories/static/js/create.js:42]: Will come back to this in a new branch.

**Pull request merged**